source/rpt/rpt.cc

```diff
@@ -38,12 +20,26 @@ RPT<dim>::calculate()
   assign_particle_positions();
   assign_detector_positions();
 
+  std::vector<double> calculated_counts;
+  std::vector<double> measured_counts;
+  if (rpt_parameters.initial_param.tuning == true)
+    {
+      measured_counts = extract_experimental_counts();
+      AssertThrow(
+        measured_counts.size() == particle_positions.size(),
+        ExcMessage(
+          "Prior tuning parameters, the number of particle positions provided"
+          " has to be the same number of counts of experimental data. "
+          "Note : The experimental counts also have to be at the same positions which can not be verified."))
```

Comment:
We need to have the same number of particle positions (position.particle) than the measured counts (counts.experimental) from experimental data since a count should be associated to a position. We can check the number but not if this is the good match.